src/functionalTest/resources/scenarios/RIA-1024-case-submitted-notification-to-case-officer-bradford.json

```diff
@@ -31,6 +35,15 @@
       }
     },
     "notifications": [
+      {
+        "reference": "1024_CASE_SUBMITTED_LEGAL_REPRESENTATIVE",
+        "recipient": "{$TEST_LAW_FIRM_A_USERNAME}",
+        "subject": "Immigration and Asylum appeal: case submitted",
+        "body": [
+          "PA/12345/2019",
+          "Talha Awan"
```

Comment:
Maybe as an enhancement for the future, you can cover here the link to the frontend and the phone number and email. There are a few scenarios doing it already. FYI.